cmd/admin-service.go

```diff
@@ -38,21 +38,14 @@ FLAGS:
   {{range .VisibleFlags}}{{.}}
   {{end}}
 COMMANDS:
-   {{range .VisibleCommands}}{{join .Names ", "}}{{ "\t" }}{{.Usage}}
-   {{end}}
+  {{range .VisibleCommands}}{{join .Names ", "}}{{ "\t" }}{{.Usage}}
```

Comment:
Not for everything is common. Each command has their own examples which need to be captured. Custom template is provided to avoid the default template in the first place.